docs/reference/intro.md

```diff
@@ -3,8 +3,34 @@ sidebar_position: 1
 id: intro
 ---
 
-# Intro
+# Welcome to OpenFeature!
 
-Welcome to OpenFeature docs!
+OpenFeature is an open standard for feature flagging, created to support a robust feature flag ecosystem using cloud native technologies.
 
-OpenFeature is an open standard for feature flag management, created to support a robust feature flag ecosystem using cloud native technologies. OpenFeature will provide a unified API and SDK, and a developer-first, cloud-native implementation, with extensibility for open source and commercial offerings.
+## What is feature flagging?
+
+Essentially, feature flags allow application behavior to be altered without a deployment of new code. In the most basic case, a feature flag that returns a boolean value can control which code path is executed based on the configured value of such a flag in the _flag management system_. The value of the flag is evaluated at runtime, meaning that the behavior of the application can be adjusted without any code changes. This can be used to decouple the release of new features from deployment.
+
+## How is this different than static configuration?
+
+Feature flags are dynamic; they are evaluated at runtime. This has a number of benefits beyond static configuration (i.e. environment variables or `.properties` files). They can be modified in near-realtime, but additionally, they can leverage runtime information to impact evaluation. Arbitrary contextual data (_evaluation context_) relevant to the application (i.e. IP address, timezone, user-identifier) can be used in _targeting rules_ to allow access to functionality, perform A/B testing, and perform progressive releases of new features. This decouples _feature-visibility logic_ from application logic, and allows feature-visibility logic to be centralized, shared, and easily managed.
+
+## How does OpenFeature fit in?
+
+OpenFeature seeks to provide a simple, consistent, and powerful API. To accomplish this, the OpenFeature SDK defines a number of flexible abstractions.
+
+### Evaluation API
+
+The _evaluation API_ is the aspect of OpenFeature that the _application author_ interacts with. It allows developers to evaluate feature flags, and use the resulting values to impact control flow or application characteristics. The evaluation API provides a framework that allows for customization of behavior and integration with various tools.
```

Comment:
Once we write them, each of these will link to a dedicated section: ie: the first mention of "evaluation API" will link to the detailed doc on the evaluation API